### ISSUE TITLE
Glyphs centered in OS X, aligned to bottom in Windows

### ISSUE THREAD

**Issue by 0x263b** [danielbruce/entypo#13]
Take a look at this to see what I mean.

http://i.imgur.com/a8Fkh.png

**Comment by aiacciu87** [comment]
I've same problem on my website, it is quite disturbing. Any solutions?

**Comment by klick** [comment]
use fontello.com to create a new version of the font. that one will be properly baselined.
see this thread: https://github.com/danielbruce/entypo/issues/8

**Comment by nuragic** [comment]
@klick ...is that the solution? to use fontello instead of entypo? 

@danielbruce please, can you update the official entypo font to fix this?

**Comment by klick** [comment]
@psikocrisis: yes.

**Comment by nuragic** [comment]
https://github.com/fontello/entypo/tree/master/font

**Comment by klick** [comment]
No, what i meant is: let fontello.com create a new version for you. That one works better. At least it did 5 moths ago.

**Comment by nuragic** [comment]
I first tried to do that, but fontello.com generated version of Entypo looks quite different from entypo.com official version (char codes doesn't match, some of the icons seems different...) so isn't true that works better. Maybe if are you starting from scratch, is better to use that.

Moreover, this version https://github.com/fontello/entypo/tree/master/font is newer, was committed 2 months ago.

**Comment by klick** [comment]
Ok. Didn’t know that.

**Comment by mrmartineau** [comment]
I had this same issue as well. Grab the .ttf from here and convert it with the Font Squirrel webfont generator making sure that the 'Fix vertical metrics' option is checked (in advanced settings). Let us know how it goes

**Comment by peterpp** [comment]
@mrmartineau: You save my day. Font Squirrel rocks. (http://www.fontsquirrel.com/tools/webfont-generator)

I get best results with "Fix Vertical Metrics" option ON and "X-height Matching" option set to "Verdana".

**Comment by mrmartineau** [comment]
@peterpp no problem dude. Interesting about the x-height-matching setting, perhaps Verdana is closest to this font...

**Comment by jreading** [comment]
+1
